JS/Lesson16/goksel/script.js

```diff
@@ -172,17 +195,182 @@ isPositiveNegativeZero(0);
 // 18.5 – 24.9	Healthy Weight
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
+let weight = 0;
```

Comment:
No need for this variable, your function takes 2 parameters - they are the only values that are needed right now